### PR TITLE
Update business card experiment configurations

### PR DESCRIPTION
**Pull request by rukizhazard**
- Images
  - 512x512: https://airocrhbi.blob.core.windows.net/ocr-data/Vertical/vdipainter/logs/business_card/super_res/2022-09-29T09-27-08_txtinpaintbizcard512_fsdp-fairscale/images/train/
  - 128x128:: https://airocrhbi.blob.core.windows.net/ocr-data/Vertical/vdipainter/logs/business_card/2022-09-20T04-14-37_txtinpaintbizcard128_fsdp-fairscale-t5xl/images/train/
- AML experiment
  - https://ml.azure.com/experiments/id/2b4624f9-701b-4daa-b4b5-98fa6c7ced77/runs/vdiptbizcard_1664443461_91040a6d?wsid=/subscriptions/b8da985a-830d-4d20-b9e5-8d4c0d798c7f/resourcegroups/FY20Vision/workspaces/OCR2&tid=72f988bf-86f1-41af-91ab-2d7cd011db47

### CODE REVIEW

File: cloud/aml_submit.py [hunk 1]
```diff
@@ -41,6 +42,8 @@ def main(
         'train_avid_sr_128x512_effinet': 'cloud/entry_avid_train_sr_128x512-effinet.sh',
         'train_avid_sr_128x512_opainet': 'cloud/entry_avid_train_sr_128x512-opainet.sh',
         'train_avid_inpaint_128_t5xxl': 'cloud/entry_avid_train_inpaint_128-t5xxl.sh',
+        'train_bizcard_inpaint_128_t5xxl': 'cloud/entry_bizcard_train_inpaint_128-t5xxl.sh',
```
Comment (taoisu) [inline cloud/aml_submit.py]: Please rename the name since it's not inpaint

File: cloud/aml_submit.py [hunk 2]
```diff
@@ -41,6 +42,8 @@ def main(
...
+        'train_bizcard_inpaint_512': 'cloud/entry_bizcard_train_inpaint_512.sh',
```
Comment (taoisu) [inline cloud/aml_submit.py]: Please rename, something like `bizcard_sr_512x4`

File: cloud/entry_bizcard_train_inpaint_128-t5xxl.sh [hunk 3]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```
Comment (taoisu) [inline cloud/entry_bizcard_train_inpaint_128-t5xxl.sh]: Please rename this file

File: cloud/entry_bizcard_train_inpaint_512.sh [hunk 4]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```
Comment (taoisu) [inline cloud/entry_bizcard_train_inpaint_512.sh]: Please rename this file

File: configs/just-diffusion/bizcard128_fsdp-fairscale-t5xl.yaml [hunk 5]
```diff
@@ -0,0 +1,69 @@
+model:
```
Comment (taoisu) [inline configs/just-diffusion/bizcard128_fsdp-fairscale-t5xl.yaml]: Please rename this file as you're not using t5xl for conditioning

Comment (taoisu) [reply configs/just-diffusion/bizcard128_fsdp-fairscale-t5xl.yaml]: maybe using fairscale-uncond

File: configs/just-diffusion/bizcard512_fsdp-fairscale.yaml [hunk 6]
```diff
@@ -0,0 +1,80 @@
+model:
```
Comment (taoisu) [inline configs/just-diffusion/bizcard512_fsdp-fairscale.yaml]: Suggest bizcard_sr_512x4-fairscale

File: scripts/load_pickle.py [hunk 7]
```diff
@@ -0,0 +1,9 @@
+import pickle
```
Comment (taoisu) [inline scripts/load_pickle.py]: Do we need this file?

**Review by taoisu — CHANGES_REQUESTED** [review]

File: scripts/load_pickle.py (hunk 7 above)
Comment (rukizhazard) [reply scripts/load_pickle.py]: May not be necessary, but for checking pickle file content? Would you prefer removing it?

File: cloud/aml_submit.py (hunk 1 above)
Comment (rukizhazard) [reply cloud/aml_submit.py]: Fixed, thank you

File: cloud/aml_submit.py (hunk 2 above)
Comment (rukizhazard) [reply cloud/aml_submit.py]: Fixed, thank you

File: cloud/entry_bizcard_train_inpaint_128-t5xxl.sh (hunk 3 above)
Comment (rukizhazard) [reply cloud/entry_bizcard_train_inpaint_128-t5xxl.sh]: Fixed, thank you

File: configs/just-diffusion/bizcard128_fsdp-fairscale-t5xl.yaml (hunk 5 above)
Comment (rukizhazard) [reply configs/just-diffusion/bizcard128_fsdp-fairscale-t5xl.yaml]: Sure, thanks

File: scripts/load_pickle.py (hunk 7 above)
Comment (rukizhazard) [reply scripts/load_pickle.py]: Removed, thanks

File: configs/just-diffusion/bizcard512_fsdp-fairscale.yaml (hunk 6 above)
Comment (rukizhazard) [reply configs/just-diffusion/bizcard512_fsdp-fairscale.yaml]: Fixed, thanks

File: cloud/entry_bizcard_train_inpaint_512.sh (hunk 4 above)
Comment (rukizhazard) [reply cloud/entry_bizcard_train_inpaint_512.sh]: Fixed, thanks

**Review by taoisu — APPROVED** [review]

**Pull request merged**